homework_2/task1.c

```diff
@@ -0,0 +1,40 @@
+#include <stdio.h>
+
+int recFib(int num)
+{
+    if (num <= 1)
+    {
+        return 1;
+    }
+    return recFib(num - 2 ) + recFib(num - 1);
```

Comment:
```suggestion
    return recFib(num - 2) + recFib(num - 1);
```
Проверяй оформление. Или auto formatter используй